CMakeLists.txt

```diff
@@ -217,14 +217,14 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
   endif(HAS_ALTIVEC)
 endif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
 
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm64")
```

Comment:
why the original PR of rocksdb doesn't have this problem?

Reply:
The developer of that PR uses an early DTK. The CPU of the DTK is different from an M1.

I don't know if it's due to system or CPU updates, however.